pom.xml

```diff
@@ -82,7 +82,7 @@
     <version.com.datastax.cassandra.cassandra-driver-core>1.0.0
     </version.com.datastax.cassandra.cassandra-driver-core>
     <version.com.drewnoakes.metadata-extractor>2.6.2</version.com.drewnoakes.metadata-extractor>
-    <version.com.fasterxml.jackson>2.9.9</version.com.fasterxml.jackson>
+    <version.com.fasterxml.jackson>2.10.1</version.com.fasterxml.jackson>
     <version.com.fasterxml.jackson.databind>2.9.10.1</version.com.fasterxml.jackson.databind>
```

Comment:
you should update also databind to corresponding minor version or not?

Reply:
I don't have an issue with upgrading databind as well. You mentioned above that databind is not synced with the rest of the Jackson groupId on releases, so I wonder if it's appropriate to bump databind version in the same PR. Again, I can bump databind if it's preferred. wdyt?

Reply:
@cuijulian I was mentioning the different versioning for jackson-databind because they release multiple times on the same micro as jackson-core just to fix CVE issues and we need to keep up with some of them, but generally if you upgrade minor and micro versions for jackson-core you should follow that version for databind too. There could be problems to not have minor and micro versioning differences.